.changeset/orange-tigers-punch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@swisspost/design-system-components-angular': patch
```

Comment:
Isn't it a major as well?

Reply:
Not sure, there is no breaking change for this package in the end.

Reply:
aria-prohibited-attr